schemas/devContainerFeature.schema.json

```diff
@@ -0,0 +1,220 @@
+          "type": "string"
+        },
+        "init": {
+          "description": "Adds the tiny init process to the container (--init) when the Feature is used.",
+          "type": "boolean"
+        },
+        "installsAfter": {
+          "description": "Array of ID’s of Features that should execute before this one. Allows control for feature authors on soft dependencies between different Features.",
+          "items": {
+            "type": "string"
+          },
+          "type": "array"
+        },
+        "licenseURL": {
+          "description": "URL to the license for the Feature.",
+          "type": "string"
+        },
+        "mounts": {
+          "items": {
+            "$ref": "#/definitions/Mount"
+          },
+          "type": "array"
+        },
+        "name": {
+          "description": "Display name of the Feature.",
+          "type": "string"
+        },
+        "options": {
+          "description": "Possible user-configurable options for this Feature. The selected options will be passed as environment variables when installing the Feature into the container.",
+          "additionalProperties": {
+            "$ref": "#/definitions/FeatureOption"
+          },
+          "type": "object"
+        },
+        "privileged": {
+          "description": "Sets privileged mode (--privileged) for the container.",
+          "type": "boolean"
+        },
+        "securityOpt": {
+          "description": "Sets container security options to include when creating the container.",
+          "items": {
+            "type": "string"
+          },
+          "type": "array"
+        },
+        "version": {
+          "description": "The version of the Feature. Follows the semanatic versioning (semver) specification.",
+          "type": "string"
+        }
+      },
+      "required": [
+        "id",
+        "version"
+      ],
+      "type": "object"
+    },
+    "FeatureOption": {
+      "additionalItems": false,
+      "anyOf": [
+        {
+          "description": "Option value is represented with a boolean value.",
+          "additionalProperties": false,
+          "properties": {
+            "default": {
+              "description": "Default value if the user omits this option from their configuration.",
+              "type": "boolean"
+            },
+            "description": {
+              "type": "string"
+            },
+            "type": {
+              "const": "boolean",
+              "type": "string"
+            }
+          },
+          "required": [
+            "type",
+            "default"
+          ],
+          "type": "object"
+        },
+        {
+          "additionalProperties": false,
+          "properties": {
+            "default": {
+              "description": "Default value if the user omits this option from their configuration.",
+              "type": "string"
+            },
+            "description": {
+              "type": "string"
```

Comment:
Add description, maybe: "A description of the option that may be shown by tools to the user."